variants/arduino_zero/linker_scripts/gcc/flash_with_bootloader.ld

```diff
@@ -67,6 +67,9 @@ SECTIONS
 	{
 		__text_start__ = .;
 
+		KEEP(*(.ota_boot))
+
+		. = ALIGN(256);
```

Comment:
I'll change it to `0x2000`.

Maybe we can have a clearer name for this section `second_stage`, `second_stage_boot` or `sketch_boot`?